packages/google_sign_in/google_sign_in/example/pubspec.yaml

```diff
@@ -23,7 +27,3 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
-
-environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.12.13+hotfix.4"
```

Comment:
I got the feeling that `environment` was misplaced in 90% of the files (maybe leaving it at the bottom would have made this PRs a little bit smaller? :P)

Reply:
💯 

But this was done deliberately to match all of the current package templates (it came up when someone from the Dart side of things was making changes and was confused by the placement). I think we decided that at some point in the past Flutter's plugin template put it at the bottom, which is why they are almost all like this.

Reply:
Thanks for trudging through these :/